src/azure-cli/azure/cli/command_modules/ams/commands.py

```diff
@@ -54,7 +54,7 @@ def get_custom_sdk(custom_module, client_factory):
         g.command('sync-storage-keys', 'sync_storage_keys')
 
     with self.command_group('ams account sp', get_sdk('Mediaservices', get_mediaservices_client)) as g:
-        g.custom_command('create', 'create_assign_sp_to_mediaservice',
+        g.custom_command('create', 'create_or_update_assign_sp_to_mediaservice',
```

Comment:
Please also update the help text to reflect the change in `create` behavior:
https://github.com/Azure/azure-cli/blob/91528ee56c70018e5a2cb56cf6527fc9b2013879/src/azure-cli/azure/cli/command_modules/ams/_help.py#L70-L72